### ISSUE TITLE
The current release doesn't work with MDAnalysis 0.16.0dev

### ISSUE THREAD

**Issue by kain88-de** [datreant/MDSynthesis#62]
trying to import mdsynthesis

```
----> 1 import mdsynthesis as mds

build/bdist.linux-x86_64/egg/mdsynthesis/__init__.py in <module>()

build/bdist.linux-x86_64/egg/mdsynthesis/treants.py in <module>()

build/bdist.linux-x86_64/egg/mdsynthesis/limbs.py in <module>()

ImportError: No module named AtomGroup
```

**Comment by richardjgowers** [comment]
Probably a MDAnalysis bug, I've raised an issue for a stub AtomGroup module

**Comment by orbeckst** [comment]
Is this fixed now, since MDAnalysis/mdanalysis#1067 was closed? Or does PR #63 fix it differently?

**Comment by kain88-de** [comment]
This one is closed the PR should still be merged though.
